src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -204,122 +255,120 @@ export class SortFilterBar
-          })}
-        </li>
-        <li>
-          ${this.getSortDisplayOption(SortField.creator, {
-            clickEvent: () => {
-              this.alphaSelectorVisible = 'creator';
-              this.selectedTitleFilter = null;
-              this.dateSortSelectorVisible = false;
-              this.setSelectedSort(SortField.creator);
-              this.emitTitleLetterChangedEvent();
-            },
-          })}
-        </li>
-      </ul>
+        <ul id="desktop-sort-selector">
+          <li>
+            ${this.showRelevance
+              ? this.getSortDisplayOption(SortField.relevance, {
+                  onClick: () => {
+                    this.clearAlphaBarFilters();
+                    this.dropdownBackdropVisible = false;
+                    this.setSelectedSort(SortField.relevance);
+                    this.emitTitleLetterChangedEvent();
+                    this.emitCreatorLetterChangedEvent();
+                  },
+                })
+              : nothing}
+          </li>
+          <li>${this.viewsDropdownTemplate}</li>
+          <li>
+            ${this.getSortDisplayOption(SortField.title, {
+              onClick: () => {
+                this.alphaSelectorVisible = 'title';
+                this.selectedCreatorFilter = null;
+                this.dropdownBackdropVisible = false;
+                this.setSelectedSort(SortField.title);
+                this.emitCreatorLetterChangedEvent();
+              },
+            })}
+          </li>
+          <li>${this.dateDropdownTemplate}</li>
+          <li>
+            ${this.getSortDisplayOption(SortField.creator, {
+              onClick: () => {
+                this.alphaSelectorVisible = 'creator';
+                this.selectedTitleFilter = null;
+                this.dropdownBackdropVisible = false;
+                this.setSelectedSort(SortField.creator);
+                this.emitTitleLetterChangedEvent();
+              },
+            })}
+          </li>
+        </ul>
+      </div>
+    `;
+  }
+
+  /** The template to render all the sort options in mobile view */
+  private get mobileSortSelectorTemplate() {
+    return html`
+      <div
+        id="mobile-sort-container"
+        class=${this.mobileSelectorVisible ? 'visible' : 'hidden'}
+      >
+        ${this.getSortDropdown({
+          displayName: html`${SortFieldDisplayName[this.selectedSort] ?? ''}`,
+          id: 'mobile-dropdown',
+          isSelected: () => true,
+          dropdownOptions: Object.keys(SortField).map(field =>
+            this.getDropdownOption(field as SortField)
+          ),
+          selectedOption: this.selectedSort ?? SortField.relevance,
+          onOptionSelected: this.mobileSortChanged,
+          onDropdownClick: () => {
+            this.dropdownBackdropVisible = this.mobileDropdown.open;
+            this.mobileDropdown.classList.toggle(
+              'open',
+              this.mobileDropdown.open
```

Comment:
SUGGESTION: Add a comma at the end of this line...?

Reply:
Our Prettier ruleset doesn't allow trailing commas on argument lists (only on multiline arrays/objects). This is probably fine since it would be an error to add additional arguments here.